diff --git a/src/game/Maps/Map.cpp b/src/game/Maps/Map.cpp
index 1276586..47086f5 100644
--- a/src/game/Maps/Map.cpp
+++ b/src/game/Maps/Map.cpp
@@ -1582,9 +1582,11 @@ void Map::UpdateObjectVisibility(WorldObject* obj, Cell cell, const CellPair& ce
 #ifdef ENABLE_PLAYERBOTS
             if (sPlayerbotAIConfig.disableBotOptimizations || player->isRealPlayer())
 #endif
-            UpdateData data;
-            player->UpdateVisibilityOf(player->GetCamera().GetBody(), obj, data);
-            data.SendData(*player->GetSession()); // TODO: This is meant to be done in next broadcast, not immediately
+            {
+                UpdateData data;
+                player->UpdateVisibilityOf(player->GetCamera().GetBody(), obj, data);
+                data.SendData(*player->GetSession()); // TODO: This is meant to be done in next broadcast, not immediately
+            }
         }
     }
 }
@@ -2707,7 +2709,7 @@ void Map::UpdateVisibility(UpdateDataMapType& update_players)
             Player* plr = m_mapRefIter->getSource();
             if (plr && plr->IsInWorld())
 #ifdef ENABLE_PLAYERBOTS
-                if (isRealPlayer())
+                if (plr->isRealPlayer())
 #endif
                     plr->HandleStealthedUnitsDetection();
         }
